content/pt-br/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -0,0 +1,586 @@
+Para fazer o teste, o kubelet executa o comando `cat /tmp/healthy` no contêiner alvo. 
+Se o comando for bem sucedido, ele retorna 0, e o kubelet considera que o contêiner está rodando 
+e íntegro. Se o comando retornar um valor não zero, o kubelet encerra o contêiner e o reinicia.
+
+Quando o contêiner inicializa, ele executa este comando:
+
+```shell
+/bin/sh -c "touch /tmp/healthy; sleep 30; rm -f /tmp/healthy; sleep 600"
+```
+
+Nos primeiros 30 segundos de execução do contêiner, existe um arquivo `/tmp/healthy`.
+Assim, durante os primeiros 30 segundos, o comando `cat /tmp/healthy` retorna um código 
+de sucesso. Depois dos 30 segundos, `cat /tmp/healthy` retorna um código de falha.
+
+Crie o Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/pods/probe/exec-liveness.yaml
+```
+
+Dentro dos 30 segundos, veja os eventos do Pod:
+
+```shell
+kubectl describe pod liveness-exec
+```
+
+A saída indica que nenhum _liveness probe_ falhou ainda:
+
+```
+Type    Reason     Age   From               Message
+  ----    ------     ----  ----               -------
+  Normal  Scheduled  11s   default-scheduler  Successfully assigned default/liveness-exec to node01
+  Normal  Pulling    9s    kubelet, node01    Pulling image "registry.k8s.io/busybox"
+  Normal  Pulled     7s    kubelet, node01    Successfully pulled image "registry.k8s.io/busybox"
+  Normal  Created    7s    kubelet, node01    Created container liveness
+  Normal  Started    7s    kubelet, node01    Started container liveness
+```
+
+Após 35 segundos, veja os eventos do Pod novamente:
+
+```shell
+kubectl describe pod liveness-exec
+```
+
+Na parte inferior da saída, existem mensagens indicando que os _liveness probes_
+falharam, e o contêiner foi encerrado e recriado.
+
+```
+  Type     Reason     Age                From               Message
+  ----     ------     ----               ----               -------
+  Normal   Scheduled  57s                default-scheduler  Successfully assigned default/liveness-exec to node01
+  Normal   Pulling    55s                kubelet, node01    Pulling image "registry.k8s.io/busybox"
+  Normal   Pulled     53s                kubelet, node01    Successfully pulled image "registry.k8s.io/busybox"
+  Normal   Created    53s                kubelet, node01    Created container liveness
+  Normal   Started    53s                kubelet, node01    Started container liveness
+  Warning  Unhealthy  10s (x3 over 20s)  kubelet, node01    Liveness probe failed: cat: can't open '/tmp/healthy': No such file or directory
+  Normal   Killing    10s                kubelet, node01    Container liveness failed liveness probe, will be restarted
+```
+
+Espere outros 30 segundos, e verifique que o contêiner foi reinicializado:
+
+```shell
+kubectl get pod liveness-exec
+```
+
+A saída mostra que `RESTARTS` foi incrementado: Note que o contador `RESTARTS` é incrementado assim que um contêiner que falhou retorne ao estado rodando:
```

Comment:
typo: ":"

"ao estado de execução" instead of "ao estado rodando" sounds better to me.